<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryStu</class>
 <widget class="QDialog" name="queryStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询学生</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cb_searchMethod">
       <item>
        <property name="text">
         <string>按姓名查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按学号查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按学院查询</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_searchContent"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="text">
        <string>开始查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
